python/tests/test_umap.py

```diff
@@ -147,9 +149,7 @@ def _run_spark_test(
 @pytest.mark.parametrize("n_parts", [2, 9])
 @pytest.mark.parametrize("n_workers", [12])
 @pytest.mark.parametrize("n_rows", [100, 500])
-@pytest.mark.parametrize(
-    "sampling_ratio", [1.0]
-)  # Temporarily set to full dataset for fit() testing
+@pytest.mark.parametrize("sampling_ratio", [0.55, 0.9])
 @pytest.mark.parametrize("supervised", [True, False])
 @pytest.mark.parametrize("dataset", ["digits", "iris"])
 @pytest.mark.parametrize("n_neighbors", [10])
```

Comment:
Please add `@pytest.mark.slow` annotation to this `test_spark_umap` parametrize test, since this test takes a long time. (See other algos for example).

Reply:
Might also want to add a simple smoke test for normal "non-slow" testing.